packages/coreutils/package.json

```diff
@@ -32,7 +32,7 @@
     "watch": "tsc -b --watch"
   },
   "dependencies": {
-    "@lumino/commands": "^1.7.0",
+    "@lumino/commands": "^1.8.0",
```

Comment:
run `jlpm run update:dependency @lumino/commands '^1.8.0'` from the root of the repo. This needs to be updated everywhere.